app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -164,9 +161,11 @@ def create
       params[:duration] = Rails.configuration.x.max_meeting_duration
     end
 
-    logger.debug("Creating meeting #{params[:meetingID]} on BigBlueButton server #{server.id}")
-    # Pass along all params except the built in rails ones
-    uri = encode_bbb_uri('create', server.url, server.secret, pass_through_params)
+    logger.debug("Creating meeting #{meeting_id} on BigBlueButton server #{server.id}")
+    callee_method = __callee__.to_s.upcase
```

Comment:
This is a bit of a mess, please don't do this metaprogramming thing with method names.

You should either have different methods to check parameters for create vs. join (you can still factor out any shared code), or have the controller method pass the list of parameters to block rather than its own name.